diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index 005caaf255529389c2684dbe8fbf0cb099d80e97..d116d730ceeb39de63dca2faac37e8337a4a8c05 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -8,6 +8,7 @@ find_library(LOG_LIB log)
 find_library(ONIG_LIB onig
     PATHS ${CMAKE_CURRENT_SOURCE_DIR}/src/main/jniLibs/${ANDROID_ABI}
     NO_CMAKE_FIND_ROOT_PATH
+    NO_DEFAULT_PATH
     REQUIRED
 )
 
